include/tvm/meta_schedule/schedule_rule.h

```diff
@@ -124,7 +125,8 @@ class ScheduleRule : public runtime::ObjectRef {
                                          bool disallow_if_then_else,  //
                                          bool require_injective,      //
                                          bool require_ordered,        //
-                                         Optional<Array<String>> disallow_op);
+                                         Optional<Array<String>> disallow_op,
+                                         Optional<Array<String>> disallow_block = {});
```

Comment:
It is less stable to specify blocks by their names, instead, please consider using other approaches to suppress inline, for example, `T.block_attr`

Reply:
hmm... Interesting idea, I didn't think of it before. Need to check

Reply:
Done. now it can be disabled through the `T.block_attr({"meta_schedule.inline_rule": "disable"})`